### PR TITLE
Pr statdump

### PR DESCRIPTION
**Pull request by iamksseo**

### CODE REVIEW

File: src/object/class_object.c [hunk 1]
```diff
@@ -2732,7 +2732,7 @@ classobj_copy_constraint_like (DB_CTMPL * ctemplate,
 
   assert (like_class_name != NULL);
 
-  assert (SM_IS_CONSTRAINT_INDEX_FAMILY (constraint_type));
+  assert (SM_IS_CONSTRAINT_INDEX_FAMILY (constraint->type));
```
Comment (pushpop0) [inline src/object/class_object.c]: if you don't need constraint_type, please delete it.

File: src/parser/compile.c [hunk 2]
```diff
@@ -163,8 +166,10 @@ pt_add_oid_to_select_list (PARSER_CONTEXT * parser, PT_NODE * statement)
       p = statement->info.query.order_by;
       while (p)
 	{
+#if !defined(NDEBUG)
 	  ord = p->info.sort_spec.expr;
 	  assert (ord->node_type == PT_VALUE);
```
Comment (pushpop0) [inline src/parser/compile.c]: if you deconstruct as shown below, you do not need variable and #if.
assert (p->info.sort_spec.expr->node_type == PT_VALUE);

File: src/object/class_object.c (hunk 1 above)
Comment (iamksseo) [reply src/object/class_object.c]: var cnstraint_type is still used;

2752   error = smt_add_constraint (ctemplate, constraint_type,

File: src/parser/compile.c (hunk 2 above)
Comment (iamksseo) [reply src/parser/compile.c]: double arrow, like X->Y->Z may  be too complex.

File: src/object/class_object.c (hunk 1 above)
Comment (pushpop0) [reply src/object/class_object.c]: modify in the same manner as 2735 line.
2752 error = smt_add_constraint (ctemplate, constraint->type,

**Pull request merged**